server/requirements.txt

```
@@ -46,8 +46,8 @@ opentelemetry-instrumentation==0.36b0 ; python_version >= "3.9" and python_versi
 opentelemetry-proto==1.15.0 ; python_version >= "3.9" and python_version < "3.13"
 opentelemetry-sdk==1.15.0 ; python_version >= "3.9" and python_version < "3.13"
 opentelemetry-semantic-conventions==0.36b0 ; python_version >= "3.9" and python_version < "3.13"
-optimum-habana==1.12.0 ; python_version >= "3.9" and python_version < "3.13"
-optimum==1.20.0 ; python_version >= "3.9" and python_version < "3.13"
+optimum-habana @ https://github.com/huggingface/optimum-habana/archive/refs/tags/ci_09082024.zip
```

```suggestion
+optimum-habana==1.13.0 ; python_version >= "3.9" and python_version < "3.13"
```
could you please update this and also rebase this PR with habana-main

---

Done.